torch/_inductor/scheduler.py

```diff
@@ -998,6 +999,51 @@ def debug_str(self) -> str:
         return "\n".join(lines).rstrip()
 
 
+# TODO<Leslie>: Find a better name
+class OuterFusedSchedulerNode(FusedSchedulerNode):
```

Comment:
This is a C++ construct, it should go in cpp.py. The other two uses of OuterFusedSchedulerNode in this file can be removed given that OuterFusedSchedulerNode inherits from FusedSchedulerNode.

Reply:
Thanks for the suggestion and changed.